scripts/postinst

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [ ! -L ${CMAKE_INSTALL_FULL_LIBDIR}/python3/dist-packages/TestHarness ]; then
+    mkdir -p ${CMAKE_INSTALL_FULL_LIBDIR}/python3/dist-packages
+    ln -s ../../../share/leap_testing/tests/TestHarness ${CMAKE_INSTALL_FULL_LIBDIR}/python3/dist-packages/TestHarness
+fi
+
+if [ ! -L ${CMAKE_INSTALL_FULL_DATAROOTDIR}/leap_testing/bin ]; then
+    mkdir -p ${CMAKE_INSTALL_FULL_DATAROOTDIR}/leap_testing
+    ln -s ../../bin ${CMAKE_INSTALL_FULL_DATAROOTDIR}/leap_testing/bin
```

Comment:
I think we ought to consider a `postrm` (or whatever the proper script would be; not super familiar with debian packaging) that cleans up the symlinks we created so they aren't left dangling after uninstalling.

Reply:
(again, I'm not super familiar with debian packaging!), I found this kind of interesting:

https://www.debian.org/doc/manuals/maint-guide/dother.en.html#links
https://manpages.debian.org/testing/debhelper/dh_link.1.en.html#FILES

is the canonical way to create these sort of links through the `package.links` file, instead of a `postinst` script?

Reply:
Not being super familiar either, but reading up on the Debian maintainer script flows, I think `scripts/prerm` might be the way to go here as well.  I'll work something up.

Reply:
> is the canonical way to create these sort of links through the `package.links` file, instead of a `postinst` script?

Yes, but ONLY if we were using debhelper to package. So, in our case, using CPack, no.

Reply:
Addressed: [be154da](https://github.com/AntelopeIO/leap/pull/818/commits/be154da6036dbe810631b53b32afee58491767fc)